packages/flutter_tools/lib/src/commands/attach.dart

```diff
@@ -45,6 +49,7 @@ final String ipv6Loopback = InternetAddress.loopbackIPv6.address;
 /// also be provided.
 class AttachCommand extends FlutterCommand {
   AttachCommand({bool verboseHelp = false, this.hotRunnerFactory}) {
+    requiresPubspecYaml();
```

Comment:
This command needs to run in the fuchsia tree and google3, so we can't require a pubspec.

Reply:
Just removed that as I saw it's breaking tests.